src/olympia/editors/models.py

```diff
@@ -734,6 +734,8 @@ def calculate_weight(self):
                 .filter(addon=addon)
                 .filter(rating__lte=3, created__gte=one_year_ago)
                 .count() * 2, 100),
+            'reputation': (
+                max(min(int(addon.reputation or 0) * -100, 300), -300)),
```

Comment:
Per the implementation plan: the reputation is multiplied by -100.

The implementation plan suggests to keep the value between -300 and 0, I've increased the max to 300 to allow for negative reputation (positive weight, moving the addon up in the queue) as suggested by @wagnerand in the issue.